common/stacks/src/test/scala/weco/api/stacks/services/SierraServiceTest.scala

```diff
@@ -77,12 +77,135 @@ class SierraServiceTest
             ontologyType = "Item"
           )
 
-          val future = service.getAccessCondition(identifier)
+          val itemNumber = SierraItemIdentifier.fromSourceIdentifier(identifier)
+
+          val future = service.getAccessConditions(Seq(itemNumber))
 
           whenReady(future) {
-            _.value shouldBe Some(
-              AccessCondition(method = AccessMethod.OnlineRequest)
+            _ shouldBe Map(
+              itemNumber ->
+                AccessCondition(method = AccessMethod.OnlineRequest)
+            )
+          }
+        }
+      }
+
+      it("if some entries are missing, returns the available AccessConditions") {
+        val responses = Seq(
+          (
+            HttpRequest(
+              uri = Uri(
+                "http://sierra:1234/v5/items?id=1601017,1234567&fields=deleted,fixedFields,holdCount,suppressed"
+              )
+            ),
+            HttpResponse(
+              entity = HttpEntity(
+                contentType = ContentTypes.`application/json`,
+                """
+                  |{
+                  |  "total": 1,
+                  |  "start": 0,
+                  |  "entries": [
+                  |    {
+                  |      "id": "1601017",
+                  |      "deleted": false,
+                  |      "suppressed": false,
+                  |      "fixedFields": {
+                  |        "79": {"label": "LOCATION", "value": "scmwf", "display": "Closed stores A&MSS Well.Found."},
+                  |        "88": {"label": "STATUS", "value": "-", "display": "Available"},
+                  |        "108": {"label": "OPACMSG", "value": "f", "display": "Online request"}
+                  |      },
+                  |      "holdCount": 0
+                  |    }
+                  |  ]
+                  |}
+                  |""".stripMargin
+              )
             )
+          )
+        )
+
+        withMaterializer { implicit mat =>
+          val service = SierraService(
+            client = new MemoryHttpClient(responses) with HttpGet
+            with HttpPost {
+              override val baseUri: Uri = Uri("http://sierra:1234")
+            }
+          )
+
+          val itemNumber = SierraItemIdentifier.fromSourceIdentifier(
+            SourceIdentifier(
+              identifierType = SierraSystemNumber,
+              value = "i16010176",
+              ontologyType = "Item"
+            )
+          )
```

Comment:
```suggestion
          val itemNumber = SierraItemNumber("1601017")
```

Isn't that equivalent but with less round tripping?

Reply:
Good spot, in this case i've just mindlessly wrapped the code with `SierraItemIdentifier.fromSourceIdentifier` when I needed a `SierraItemNumber`